gui/default/index.html

```diff
@@ -344,118 +344,142 @@ <h3 class="panel-title">
-          </div>
+        <div class="panel-group" id="login-options">
 
-          <div class="form-group">
-            <label for="password" translate>Password</label>
-            <input id="password" class="form-control" type="password" name="password" ng-model="login.password" ng-trim="false" autocomplete="current-password" />
-          </div>
+          <div class="panel panel-default">
+            <button class="btn panel-heading" data-toggle="collapse" data-parent="#login-options" data-target="#login-option-password" aria-expanded="{{isDefaultLoginOption('password') ? 'true' : 'false'}}">
+              <h4 class="panel-title">
+                <div class="panel-title-text" translate>Password</div>
+              </h4>
+            </button>
+            <div id="login-option-password" class="panel-collapse collapse {{isDefaultLoginOption('password') ? 'in' : ''}}">
+              <div class="panel-body">
+                <form ng-submit="authenticatePassword()">
+                  <div class="form-group">
+                    <label for="user" translate>User</label>
+                    <input id="user" class="form-control" type="text" name="user" ng-model="login.username" autofocus required autocomplete="username" />
+                  </div>
 
-          <div class="form-group">
-            <label>
-              <input type="checkbox" ng-model="login.stayLoggedIn" >&nbsp;<span translate>Stay logged in</span>
-            </label>
-          </div>
+                  <div class="form-group">
+                    <label for="password" translate>Password</label>
+                    <input id="password" class="form-control" type="password" name="password" ng-model="login.password" ng-trim="false" autocomplete="current-password" />
+                  </div>
 
-          <div class="row">
-            <div class="col-md-9 login-form-messages">
-              <p ng-if="login.errors.badLogin" class="text-danger" translate>
-                Incorrect user name or password.
-              </p>
-              <p ng-if="login.errors.failed" class="text-danger" translate>
-                Login failed, see Syncthing logs for details.
-              </p>
-            </div>
-            <div class="col-md-3 text-right">
-              <button type="submit" id="submit" class="btn btn-default" ng-disabled="login.inProgress" translate>Log In</button>
+                  <div class="form-group">
+                    <div class="col-md-9">
+                      <label class="stay-logged-in">
+                        <input type="checkbox" ng-model="login.stayLoggedIn" >&nbsp;<span translate>Stay logged in</span>
+                      </label>
+                    </div>
+                    <div class="col-md-3 text-right">
+                      <button type="submit" id="submit" class="btn btn-primary" ng-disabled="login.inProgress" translate>
+                        Log In
+                      </button>
```

Comment:
No wrap, as it was on main.

Reply:
Thanks, done in 00ba31eb965e286e0f1c468cd8c04d83cbb8f3d9.